cypress/support/config/services.js

```diff
@@ -309,341 +480,512 @@ export default {
-      errorPage404:
-        Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
-          ? undefined
-          : '/pashto/articles/c123456abcdo',
-      frontPage: Cypress.env('APP_ENV') === 'live' ? undefined : '/pashto',
+      articles: {
+        path:
+          Cypress.env('APP_ENV') === 'live'
+            ? '/pashto/articles/c70970g2251o'
+            : '/pashto/articles/cng0e8v85eko',
+        smoke: false,
+      },
+      errorPage404: {
+        path:
+          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
+            ? undefined
+            : '/pashto/articles/c123456abcdo',
+        smoke: false,
+      },
+      frontPage: {
+        path: Cypress.env('APP_ENV') === 'live' ? undefined : '/pashto',
+        smoke: false,
+      },
     },
   },
   persian: {
     font: 'Nassim',
     isWorldService: true,
     pageTypes: {
-      articles:
-        Cypress.env('APP_ENV') === 'live'
-          ? '/persian/articles/c7eel0lmr4do'
-          : '/persian/articles/c4vlle3q337o',
-      errorPage404:
-        Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
-          ? undefined
-          : '/persian/articles/cxvxrj8tvppo',
-      frontPage: Cypress.env('APP_ENV') === 'live' ? undefined : '/persian',
+      articles: {
+        path:
+          Cypress.env('APP_ENV') === 'live'
+            ? '/persian/articles/c7eel0lmr4do'
+            : '/persian/articles/c4vlle3q337o',
+        smoke: true,
+      },
+      errorPage404: {
+        path:
+          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
+            ? undefined
+            : '/persian/articles/cxvxrj8tvppo',
+        smoke: true,
+      },
+      frontPage: {
+        path: Cypress.env('APP_ENV') === 'live' ? undefined : '/persian',
+        smoke: false,
+      },
     },
   },
   pidgin: {
     font: undefined,
     isWorldService: true,
     pageTypes: {
-      articles:
-        Cypress.env('APP_ENV') === 'live'
-          ? undefined
-          : '/pidgin/articles/cwl08rd38l6o',
-      errorPage404:
-        Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
-          ? undefined
-          : '/pidgin/articles/cxvxrj8tvppo',
-      frontPage: '/pidgin',
+      articles: {
+        path:
+          Cypress.env('APP_ENV') === 'live'
+            ? undefined
+            : '/pidgin/articles/cwl08rd38l6o',
+        smoke: false,
+      },
+      errorPage404: {
+        path:
+          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
+            ? undefined
+            : '/pidgin/articles/cxvxrj8tvppo',
+        smoke: true,
+      },
+      frontPage: { path: '/pidgin', smoke: true },
     },
   },
   portuguese: {
     font: undefined,
     isWorldService: true,
     pageTypes: {
-      articles: undefined,
-      errorPage404: undefined,
-      frontPage: undefined,
+      articles: { path: undefined, smoke: false },
```

Comment:
Why does this not have an article path and a 404 path? It's OOS, but is this is a known issue?

Reply:
Ah I might have confused it for a pending one because of the portuguese/brasil logic, ile add it now

Reply:
Added frontpage and article, https://www.test.bbc.com/portuguese/articles/cerj3rk9mmyo is 404'ing for some reason, dispite https://www.test.bbc.com/portuguese/articles/cerj3rk9mmyo.json working through... any ideas @hindsc52?